### PR TITLE
Fix ppo

### PR DESCRIPTION
**Pull request by shagunsodhani**
Fixes https://github.com/facebookresearch/rl/issues/92

### CODE REVIEW

**Comment by vmoens** [comment]
Thanks for pointing and solving this @shagunsodhani! 
I should integrate tests of the training scripts in the CI but it is not yet clear to me how that could be done efficiently, and I don't test them all every time I commit something...
I'll prioritize that!

**Review by vmoens — APPROVED** [review]

File: torchrl/objectives/costs/ppo.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def __init__(
     def forward(self, tensordict: _TensorDict) -> _TensorDict:
         if self.advantage_module is not None:
             tensordict = self.advantage_module(tensordict)
-        tensordict = tensordict.clone()
+        tensordict = tensordict.clone().detach()
```
[the remark this answers is not in the document]
Comment (shagunsodhani) [reply torchrl/objectives/costs/ppo.py:220]: It does break in the current form - error log is in the [issue](#92 )

[the remark this answers is not in the document]
Comment (vmoens) [reply torchrl/objectives/costs/ppo.py:220]: I have checked out a [new PR](#95) with the fix, hope that solves it!

**Comment by vmoens** [comment]
Closing as per #95

**Pull request closed without merging**